[12.0][IMP] Improvement in hotel reservation module

vimalpatelserpentcs opened the pull request.

dhruvitaSerpentCS committed:
[IMP]hotel_reservation:optimized code

vimalpatelserpentcs committed:
Merge branch '12.0-mig-hotel-reservation' of https://github.com/vimalpatelserpentcs/vertical-hotel into 12.0-mig-hotel-reservation

vimalpatelserpentcs committed:
[IMP]hotel_reservation:Make code as per OCA standard

JayVora-SerpentCS commented on hotel_reservation/__manifest__.py:
```diff
@@ -1,30 +1,30 @@
+    "author": "Odoo Community Association (OCA), Serpent Consulting \
+                Services Pvt. Ltd., Odoo S.A.",
+    "category": "Generic Modules/Hotel Reservation",
+    "license": "AGPL-3",
```
Add a website parameter please

vimalpatelserpentcs replied:
Done

JayVora-SerpentCS commented on hotel_reservation/models/hotel_reservation.py:
```diff
@@ -112,70 +113,117 @@ class HotelReservation(models.Model):
+        string="Folio",
+    )
+    no_of_folio = fields.Integer("No. Folio", compute="_compute_folio_id")
+    dummy = fields.Datetime("Dummy")
```
Any idea why this field was added?

vimalpatelserpentcs replied:
Actually, there is no major use of these fields but only use in the calendar view for display, but now we have removed it and in a calendar view we have added check-in and  check-out dates

JayVora-SerpentCS commented on hotel_reservation/models/hotel_reservation.py:
```diff
@@ -195,99 +243,113 @@ def unlink(self):
+                        "You cannot delete Reservation in %s\
+                                         state."
+                    )
+                    % (reserv_rec.state)
```
Simplify this : "Sorry, you can only delete the reservation when it's draft!"

vimalpatelserpentcs replied:
Done

vimalpatelserpentcs committed:
[IMP]hotel_reservation: improvement the code and resolved the conversion too.

vimalpatelserpentcs committed:
[IMP]hotel_reservation: fix the travis

vimalpatelserpentcs committed:
[FIX]hotel_reservation:fixed the travis

[FIX]hotel_reservation:fixed the js file conflict

[FIX]hotel_reservation: Fixed the travis error

vimalpatelserpentcs committed:
[FIX]hotel_reservation: fixed the reservation summary issue
